### DISCUSSION TITLE
Editing an item field (e.g., the title) in list view possible?

### DISCUSSION THREAD

**Discussion by wq9578** [filamentphp/filament#10470] [Help]
Hi, I'm new to Filament and only did a first successful test, which was quite impressive.
Is it possible to edit, for example, the title of an item (article name) in the list view?
I would like to edit the item titles (or another field displayed in the list view) without having to open/click every single item individually.

**Comment by ArtMin96** [comment]
[Text input column](https://filamentphp.com/docs/3.x/tables/columns/text-input)

**Comment by wq9578** [comment]
Thanks, great!

Is it possible to make the frame around the text input invisible?

I would like to use some formatting (text color, or italics/non-italics) to indicate editable/non-editable text.

**Reply by ArtMin96** [reply]
Use CSS?

ex.
```php
TextInputColumn::make('email')
    ->extraAttributes(['class' => 'some-class'])
    // Or
    ->extraInputAttributes(['class' => 'some-class'])
```

```css
.some-class {
    @apply ...;
}
```

**Comment by wq9578** [comment]
Thank you very much!
I will try this.
